phpcs.xml.dist

```diff
@@ -21,11 +21,33 @@
         <exclude name="SlevomatCodingStandard.Classes.DisallowLateStaticBindingForConstants.DisallowedLateStaticBindingForConstant"/>
     </rule>
 
-    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint">
+    <!-- See https://github.com/squizlabs/PHP_CodeSniffer/issues/2937 -->
+    <rule ref="Squiz.Arrays.ArrayDeclaration.ValueNoNewline">
+        <exclude-pattern>*/lib/*</exclude-pattern>
+        <exclude-pattern>*/tests/*</exclude-pattern>
+    </rule>
+
+    <rule ref="Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps">
+        <!-- This test contains a fixture class that should use snake case  -->
+        <exclude-pattern>tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php</exclude-pattern>
+    </rule>
+    <rule ref="Squiz.NamingConventions.ValidVariableName.NotCamelCaps">
+        <!-- The dumper has special variables, and the test contains references to print_r  -->
+        <exclude-pattern>lib/Doctrine/DBAL/Tools/Dumper.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/DBAL/Tools/DumperTest.php</exclude-pattern>
+        <!-- This files uses the return value db2_server_info(), which does not follow conventions  -->
```

Comment:
If something like the usage of `$server->DBMS_VER` is flagged, then it's a bug in the standard. The caller cannot change the API being called. The naming convention is only violated by the defining side.

Reply:
I just reported this. I will improve the comment with a link to the bug report.